src/insert.rs

```diff
@@ -28,19 +28,27 @@ const MIN_CHUNK_SIZE: usize = BUFFER_SIZE - 1024; // slightly less to avoid extr
 /// Rows are being sent progressively to spread network load.
 #[must_use]
 pub struct Insert<T> {
+    client: Client,
+    table: String,
+    fields: String,
```

Comment:
I think this should be removed from the Insert itself, but I could not make it work in the `init_request` method because of the `trait bound is not satisfied` error, as the compiler thinks `T` is not `Row` there; but it works fine in the `new` method.

Reply:
Yes, because `write()` and `do_write()` doesn't require `T: Row` for now.